SECURITY.md

```diff
@@ -0,0 +1,62 @@
+# Security
+
+As part of our vulnerability disclosure policy, we operate a security vulnerability program through [Immunefi](https://immunefi.com/). This document serves as a complementary guideline for reporting vulnerabilities and how the disclosure process is managed. Please refer to the official Evmos [bug bounty program](https://immunefi.com/bounty/evmos/) for up-to-date information.
```

Comment:
The bug bounty program link (https://immunefi.com/bounty/evmos/) is returning 404

Reply:
Good pointer! But this is because we havent finalized some details yet - It should be live soon!